ui/app/src/health-client.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The Perses Authors
```

Comment:
Would it make sense to put this file into the `core/src/hooks` dir along with the other hooks in there, or is this not "core"-specific enough? Not 100% sure in general what should go into `core` vs. the `app`, I guess we'll be figuring that out :)

Reply:
and I'm fine with this approach. At least it's simpler \o/

Reply:
Ah ok, fine by me :)

Reply:
Yeah, I can talk you through the philosophy I usually follow with data-fetching stuff like this maybe next time we get together. Might even be worth putting it into a GitHub discussion. For `app` vs `core`, I'm thinking "do plugins need this too" and if so, it goes in `core`, otherwise starts in `app`.

Once you decide which project/package, then it's a question of "where in the package do I put it", so maybe the more interesting question here is "where in app does it go"? And I think that's an open question (I have thoughts), but if we're going to potentially reorganize the folder structure, may have a different answer once we figure that out. 😂